src/libPersistence/Retriever.cpp

```diff
@@ -256,8 +256,16 @@ bool Retriever::RetrieveBlockLink(bool trimIncompletedBlocks) {
         LOG_GENERAL(WARNING, "Get LatestActiveDSBlockNum failed");
         return false;
       }
-      m_mediator.m_ds->m_latestActiveDSBlockNum = std::stoull(
-          DataConversion::CharArrayToString(latestActiveDSBlockNumVec));
+
+      auto dsBlockNumStr =
+          DataConversion::CharArrayToString(latestActiveDSBlockNumVec);
+      try {
```

Comment:
The two exceptions thrown by `stoull` can be caught by `std::exception`. There is no need to use elipses.

Reply:
Why is the try.. catch for this stoull? I cannot find it...

Reply:
Ok, i see. I fixed with rebase the commit, please review again. @KaustubhShamshery